app/controllers/transactions_controller.rb

```diff
@@ -45,13 +51,7 @@ def in_review
     order_details = current_user.administered_order_details.in_review
 
     @search_form = TransactionSearch::SearchForm.new(params[:search])
-    @search = TransactionSearch::Searcher.new(TransactionSearch::FacilitySearcher,
-                                              TransactionSearch::AccountSearcher,
-                                              TransactionSearch::ProductSearcher,
-                                              TransactionSearch::DateRangeSearcher,
-                                              TransactionSearch::OrderStatusSearcher,
-                                              TransactionSearch::AccountOwnerSearcher,
-                                              TransactionSearch::OrderedForSearcher).search(order_details, @search_form)
+    @search = TransactionSearch::Searcher.new(searchers).search(order_details, @search_form)
```

Comment:
This might need to be `TransactionSearch::Searcher.new(
*searchers)`, but maybe the initializer is smart enough to deal with either type of input.